src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -79,25 +80,40 @@ public void SetAndCheckBasePriority(ProcessPriorityClass exPriorityClass, int pr
             Assert.Equal(priority, _process.BasePriority);
         }
 
-        [Fact]
+        [Fact, PlatformSpecific(PlatformID.Windows)]
         public void Process_BasePriority()
         {
             ProcessPriorityClass originalPriority = _process.PriorityClass;
             Assert.Equal(ProcessPriorityClass.Normal, originalPriority);
 
-            if (global::Interop.IsWindows)
+            try
             {
-                try
-                {
-                    //SetAndCheckBasePriority(ProcessPriorityClass.RealTime, 24);
-                    SetAndCheckBasePriority(ProcessPriorityClass.High, 13);
-                    SetAndCheckBasePriority(ProcessPriorityClass.Idle, 4);
-                    SetAndCheckBasePriority(ProcessPriorityClass.Normal, 8);
-                }
-                finally
-                {
-                    _process.PriorityClass = originalPriority;
-                }
+                //SetAndCheckBasePriority(ProcessPriorityClass.RealTime, 24);
+                SetAndCheckBasePriority(ProcessPriorityClass.High, 13);
+                SetAndCheckBasePriority(ProcessPriorityClass.Idle, 4);
+                SetAndCheckBasePriority(ProcessPriorityClass.Normal, 8);
+            }
+            finally
+            {
+                _process.PriorityClass = originalPriority;
+            }
+        }
+
+        [Fact, PlatformSpecific(PlatformID.AnyUnix), OuterLoop] // This test requires admin elevation on Unix
```

Comment:
I'm wondering whether OuterLoop is really the right category for things that require elevated privileges.  Is that how we've defined it?  Or do we need a separate category for things that require such privileges?  I often run OuterLoop tests locally, because I want to make sure I've run all relevant tests before submitting a PR, and thus far I haven't been doing it elevated... with this I'll need to start doing so.  If that's the path we're on, ok, but we should probably be clear somewhere about what it means.